Lib/glyphsLib/builder.py

```diff
@@ -340,6 +344,10 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name.startswith('openTypeOS2Win') and value < 0:
             value = -value
 
+        # Some Glyphs files store weightClass/widthClass as string values.
+        if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
+            value = int(value)
```

Comment:
Actually, even if we add `'weightClass'` and `'widthClass'` to `CUSTOM_INT_PARAMS` set, we still have to ensure that `value = int(value)` inside the `set_custom_params` function.

That's because after https://github.com/googlei18n/glyphsLib/pull/120 we read custom parameters as generic numbers: so if their value happen to be a float (it shouldn't in this case, but Glyphs.app allows that as it does not enforce integers in custom parameters), then we would read them as float (or generic `num`).
However, for the UFO spec, the `openTypeOS2{Weight,Width}Class` must be integers, so we must make them so when setting these in the builder, if they're not already.